rca/landingpages/models.py

```diff
@@ -291,13 +306,34 @@ class LandingPage(TapMixin, ContactFieldsMixin, LegacyNewsAndEventsMixin, BasePa
             heading=_("Related pages grid"),
         ),
         InlinePanel("featured_image", label=_("Featured content"), max_num=1),
-        FieldPanel("legacy_news_and_event_tags"),
+        MultiFieldPanel(
+            [
+                HelpPanel(
+                    content=(
+                        """<p>The title, link and link text displayed as part of the news and events
+                        listing can be customised by adding overriding values here</p>"""
+                    )
+                ),
+                FieldPanel("news_and_events_title"),
+                FieldPanel("news_and_events_link_text"),
+                FieldPanel("news_and_events_link_target_url"),
+                FieldPanel("legacy_news_and_event_tags"),
+            ],
+            "News and Events",
+        ),
         MultiFieldPanel(
             [FieldPanel("page_list_title"), StreamFieldPanel("page_list")],
             heading=_("Related page list"),
         ),
     ]
 
+    @property
+    def news_view_all(self):
+        return {
+            "link": self.news_and_events_link_target_url,
+            "title": self.news_and_events_link_text,
+        }
+
```

Comment:
Suggestion - there's quite a bit of repetition with the field definitions, helper property and content panel definitions. Could we refactor this into a mixin class? (the content panels could be returned from a method that takes extra `FieldPanels` as arguments to get around the `legacy_news_and_event_tags` not being on all Page types)